src/js/selection.js

```diff
@@ -311,6 +312,7 @@
             while (!stop && node) {
                 // Only iterate over elements and text nodes
                 if (node.nodeType > 3) {
+                    node = nodeStack.pop;
```

Comment:
Yes. Not sure how that happened.. must have accidentally backspaced when I was searching through my file before saving. Will fix now.